python/kwiver/vital/config/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ kwiver_add_python_library( config
 target_link_libraries( python-vital.config-config
   PRIVATE              ${PYTHON_LIBRARIES}
                        vital_config
+                       vital
```

Comment:
Here vital and vital_config should probably be PUBLIC since they are included in config.h, thus exposed through the public API.